src/app/course-home-page/footer/footer.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.css']
+})
+export class FooterComponent implements OnInit {
+  toggle=true;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onToggle(){
+      console.log(this.toggle)
+    if(this.toggle===true){
```

Comment:
there is a better way to toggle. It is synthetic,  if else case is not required as you are always going to toggle. 
`this.toggle = !this.toggle` should wrap this up.